arches/app/templates/javascript.htm

```diff
@@ -326,7 +324,6 @@
         date-format='{% trans "Date Format" as dateFormat %} "{{ dateFormat|escapejs }}"'
         has-no-value='{% trans "Has no value" as hasNoValue %} "{{ hasNoValue|escapejs }}"'
         has-any-value='{% trans "Has any value" as hasAnyValue %} "{{ hasAnyValue|escapejs }}"'
-        default-value='{% trans "Default Value" as defaultValue %} "{{ defaultValue|escapejs }}"'
```

Comment:
Still seems to be in use in several places:
https://github.com/archesproject/arches/blob/a1c2b429f5aaa54af95a4111862eb87d56709107/arches/app/templates/views/components/widgets/datepicker.htm#L83-L85

Reply:
These were dupes.